code-gen-projects/typescript/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "ion-cli-typescript-demo",
+  "version": "1.0.0",
+  "description": "TypeScript code generation demo for ion-cli",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "prebuild": "npm run generate",
+    "build": "tsc",
+    "test": "jest",
+    "generate": "ion-cli generate -l typescript -d ../../schema -o ./src/models",
```

Comment:
This should be `ion` instead of `ion-cli` since thats the executable name used in the CLI tool.